File: tests/fixtures/elastic-ts/basic/no_imports.ts
// File without @elastic/elasticsearch import — must produce zero
// emits even though `client.search(...)` shape matches.

const fakeClient = {
  search(_opts: { index: string }) {
    return Promise.resolve();
  },
};

export function local() {
  return fakeClient.search({ index: 'not-elastic' });
}

File: tests/fixtures/elastic-rs/basic/src/no_imports.rs
// File without elasticsearch crate use — must produce zero emits.

pub struct FakeStore;

impl FakeStore {
    pub async fn index(&self, _opts: ()) {}
    pub async fn search(&self, _opts: ()) {}
}

pub async fn local(s: &FakeStore) {
    s.index(()).await;
    s.search(()).await;
}
